ui/src/app/store/project/project.action.ts

```diff
@@ -0,0 +1,34 @@
+
+import { LoadOpts, Project } from 'app/model/project.model';
+
+// Use to load fetched Project in our app
+export class LoadProject {
```

Comment:
LoadProjectAction

Reply:
No, try to be concise as possible and follow guidelines https://ngxs.gitbook.io/ngxs/recipes/style-guide#action-suffixes